Building-with-IntelliJ-IDEA.md

```diff
@@ -156,7 +156,9 @@ If you've followed all instructions correctly, try deleting the `%TEMP%/cache-16
 
 ## Pluginhub failing to load
 
-This happens because the development versions are snapshot versions, whereas we only provide builds of pluginhub plugins for stable releases of RuneLite. Usually you can pass `-Drunelite.pluginhub.version=<current RL version>` as a VM option to load the plugins built against the release into your build, which will work if there are no ABI changes. If there are ABI changes, build the plugins yourself and place them into the `sideloaded-plugins` directory when running in developer mode.
+External plugins are not installed by default in development versions of RuneLite because the development versions are snapshot versions, whereas we only provide builds of pluginhub plugins for stable releases of RuneLite. Usually you can pass `-Drunelite.pluginhub.version=<current RL version>` as a VM option (to enable VM Options, follow the guide in [[Using the client developer tools]]) to load the plugins built against the release into your build, which will work if there are no [Application Binary Interface](https://en.wikipedia.org/wiki/Application_binary_interface) (ABI) changes.
```

Comment:
how about something like "Hub plugins are not available by default when developing with snapshot versions, as they are only built for stable releases of RuneLite." for the first sentence.  I've never liked calling them 'external plugins', and cuts down the length.

Reply:
Sure thing, I like that. I'll update it real quick.